docs/changelog-fragments/756.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+The ``Channel`` class calls to libssh ``ssh_channel_open_session`` and
```

Comment:
I'm pretty sure classes don't call things. They represent states. Could you rephrase?